pkg/apis/build/v1alpha2/build_pod_test.go

```diff
@@ -2757,3 +2990,17 @@ func fetchEnvVar(envVars []corev1.EnvVar, name string) (corev1.EnvVar, bool) {
 
 	return corev1.EnvVar{}, false
 }
+
+func or(bools ...bool) bool {
+	for _, b := range bools {
+		if b {
+			return true
+		}
+	}
+
+	return false
+}
+
+func boolPointer(b bool) *bool {
```

Comment:
Is there a place for "helper" functions to live, to avoid [redeclaration](https://github.com/pivotal/kpack/blob/4be6c3d36dd19025ca245a9e91f338526f8bdbbb/pkg/apis/build/v1alpha2/build_pod.go#L667)?

Reply:
I don't believe there is a specific kpack wide place for helpers. we have some helpers in various packages they are used in.